DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/AddCentreCustomPromptsAccessibilityTests.cs

```diff
@@ -18,20 +18,30 @@ public void AddRegistrationPrompt_journey_has_no_accessibility_errors()
 
             // when
             Driver.Navigate().GoToUrl(BaseUrl + startUrl);
+            AnalyzePageHeading("Add delegate registration prompt");
             var selectPromptResult = new AxeBuilder(Driver).Analyze();
             Driver.SelectDropdownItemValue("CustomPromptId", "1");
             Driver.SubmitForm();
 
+            AnalyzePageHeading("Configure answers");
             var configureAnswerInitialResult = new AxeBuilder(Driver).Analyze();
+
             AddAnswer("Answer 1");
             AddAnswer("Answer 2");
+            AnalyzePageHeading("Configure answers");
             var configureAnswerWithAnswersResult = new AxeBuilder(Driver).Analyze();
+
             Driver.ClickButtonByText("Next");
 
+            AnalyzePageHeading("Summary");
+            var summaryResult = new AxeBuilder(Driver).Analyze();
+            Driver.SubmitForm();

```

Comment:
Hmm.. Could we skip the "SubmitForm" step so we don't keep on adding the new prompts into the test db?